src/components/ConfigEditor.tsx

```diff
@@ -67,7 +67,10 @@ export const ConfigEditor: React.FC<Props> = (props: Props) => {
 
   useEffect(() => {
     setDisabled(true);
-    if (options.jsonData.baseUrl && (options.secureJsonFields?.accessToken || options.secureJsonData?.accessToken)) {
+    if (
+      (options.jsonData.baseUrl && (options.secureJsonFields?.accessToken || options.secureJsonData?.accessToken)) ||
```

Comment:
This could be a little clearer IMO, nested expressions in the middle of a multi-conditional expression don't really scan well.